### PR TITLE
fix: resolve import issues

### PR DESCRIPTION
**Pull request by DanielRuf**
This PR resolves the issues with the wrong imports, as these are not modules but cjs.

### CODE REVIEW

File: src/core/infrastructure/KyRequester.ts [hunk 1]
```diff
@@ -26,7 +25,7 @@ function defaultRequest(service: any, { body, query, sudo, method }) {
   if (sudo) headers.append('sudo', `${sudo}`);
 
   if (typeof body === 'object' && !(body instanceof FormData)) {
-    bod = JSON.stringify(decamelizeKeys(body, skipAllCaps));
+    bod = JSON.stringify(decamelizeKeys(body));
```
Comment (jdalrymple) [inline src/core/infrastructure/KyRequester.ts]: Does it avoid decamelizing if the input is in the form of "SOME_UPPER_CASE_VARIABLE" ?

Comment (DanielRuf) [reply src/core/infrastructure/KyRequester.ts]: Screenshot_20191022-050808

**Comment by jdalrymple** [comment]
@DanielRuf any suggestions for an xcase replacement? Though it says it supports "some-string" to camelcase, it doesn't seem to function properly https://runkit.com/embed/tlx14hnn6mux

**Pull request merged**